bin/cmds/create.js

```diff
@@ -9,67 +9,64 @@ const fs = require('fs')
 const fabricator = require('../../lib')
 const bistre = require('bistre')()
 
-exports.command = 'create [options]'
```

Comment:
@aearly this was the only thing I wasn't 100% on.  I couldn't see this being utilized anywhere but you never know.